lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -169,13 +150,15 @@ function dosomething_rogue_get_by_file_id($fid)
  * phoenix and it's corresponding id's in Rogue
  *
  * @param string $rbid
- * @param string $fid
  * @param object $rogue_reportback
  *
  * @return InsertQuery object
  */
-function dosomething_rogue_store_rogue_references($rbid, $fid, $rogue_reportback)
+function dosomething_rogue_store_rogue_references($rbid, $rogue_reportback)
 {
+  $reportback = entity_load_unchanged('reportback', [$rbid]);
+  $fid = array_pop($reportback->fids);
```

Comment:
hmm I'm wondering if this part should get held off on and then there will be fewer conflicts with my PR? I handled this part differently with `dosomething_rogue_check_rbid_and_store_ref` https://github.com/DoSomething/phoenix/pull/7224/files

what do you think would be the best way to proceed?